### ISSUE TITLE
render widget with Panel library

### ISSUE THREAD

**Issue by nlooije** [deathbeds/ipydrawio#86]
Sorry for the many bug reports and feature requests lately. 
I really appreciate the work you put into ipydrawio and any feedback and support you give.

## Motivation

I would like for the ipydrawio widget to be able to render within a [Panel](https://panel.holoviz.org/) app.
> Panel is an open-source Python library that lets you create custom interactive web apps and dashboards by connecting user-defined widgets to plots, images, tables, or text.

This library is useful when integrating the ipydrawio widget into a GUI

> Panel library can be used to develop rich interactive applications without tying your domain-specific code to any particular GUI or web tools.

## Design Ideas

Currently, it doesn't seem that the ipydrawio widget renders within the Panel library.

![image](https://user-images.githubusercontent.com/3252193/148347060-d6a085ee-b807-48c6-ab39-ed0c22984aee.png)

No errors are generated, nothing to help make it render or where to start:

![image](https://user-images.githubusercontent.com/3252193/148348874-6a406550-fb3f-49bc-b3fd-533799f95457.png)

Rather than issuing a bug report i figured that rendering within Panel wasn't a design goal so it would be best to add this as a feature request.

Honestly, i don't have the faintest idea why it is not rendering with panel. Other widgets are rendering something:

**ipyleaflet:**
![image](https://user-images.githubusercontent.com/3252193/148349307-3cbd8ac8-a821-4ec1-a228-71ae3f19b865.png)

**ipyvolume:**
![image](https://user-images.githubusercontent.com/3252193/148349355-22a420bf-96b1-48ac-b2b0-ed641d6a206b.png)

**Comment by bollwyvl** [comment]
> without tying your domain-specific code to any particular GUI or web tools.

>> ... except panel

Yeah, panel integration is kind of a moving target, non-goal, and a dependency unacceptable, but if you contributed a tutorial as a PR with tests, I'd take it. You'll probably want to dig a bit deeper into the DOM itself to see if it's actually showing up... for... reasons... under the hood `Diagram` is actually a [Box](https://github.com/deathbeds/ipydrawio/blob/master/packages/ipydrawio/src/widgets.ts#L109) and might just need a bit more `layout` configuration to appease whatever panel wraps it in. It may be easier to experiment with _another_ box around it, and if panel can't handle Boxes, then that's pretty much endgame.

**Comment by nlooije** [comment]
Ok, Panel seems to handle boxes just fine:

![image](https://user-images.githubusercontent.com/3252193/148531841-d51949b4-e552-42d4-a713-afbe2fa3ad31.png)

and wrapping `Diagram` in a box also gives no issues:

![image](https://user-images.githubusercontent.com/3252193/148531949-c1423480-423b-4ba2-a42a-6b6b7144f196.png)

but combining yields nothing:

![image](https://user-images.githubusercontent.com/3252193/148532055-a8ff420c-2da5-4b7e-b084-43470c693794.png)

I am still looking at what the main differences in HTML generated are
